qa/L0_python_api/test_model_repository/identity/config.pbtxt

```diff
@@ -30,14 +30,14 @@ max_batch_size: 0
 
 input [
   {
-    name: "INPUT0"
+    name: "text_input"
```

Comment:
Why changing the IO names?

Reply:
Changed the IO names to make the model compatible with the Triton CLI.

Reply:
I would recommend running the general pipeline + L0_backend_python to see if any tests depend on input names